app/views/shared/_part.html.erb

```diff
@@ -17,6 +17,9 @@
           <span class="form-wrapper">
             <%= f.text_field :title, disabled: @resource.locked_for_edits?, class: "title form-control" %>
           </span>
+          <% if f.object.errors[:title].any? %>
+            <%= form_errors(f.object.errors[:title]) %>
+          <% end %>
```

Comment:
As per https://github.com/alphagov/publisher/pull/1488#discussion_r689405171 convention dictates that this shouldn't have the conditional check

Reply:
Fixed - thanks